docker-compose.yml

```diff
@@ -39,7 +40,7 @@ services:
       - ./app-frontend/package.json:/opt/raster-foundry/app-frontend/package.json
       - ./app-frontend/src:/opt/raster-foundry/app-frontend/src
       - ./app-frontend/webpack.config.js:/opt/raster-foundry/app-frontend/webpack.config.js
-    command: npm run build-watch
+    command: npm run start
```

Comment:
Hm. This may be a result of anyone using JavaScript tooling outside of the virtual machine to work on frontend stuff. Having it in a working state is the way to go regardless. 👍